pod/pod.go

```diff
@@ -74,3 +73,21 @@ func (s *Service) Delete(ctx context.Context, name string) error {
 
 	return nil
 }
+
+// DeleteAllPod deletes all pods.
+func (s *Service) DeleteAll(ctx context.Context) error {
+	if err := s.client.CoreV1().Pods(defaultNamespaceName).DeleteCollection(ctx, metav1.DeleteOptions{}, metav1.ListOptions{}); err != nil {
+		return fmt.Errorf("delete all pods: %w", err)
+	}
+	return nil
+}
+
+// DelteAllScheduledPod deletes all scheduled pods.
+func (s *Service) DeleteAllScheduledPod(ctx context.Context) error {
```

Comment:
nits: DeleteAllScheduledPod → DeleteAllScheduledPods
```suggestion
// DeleteAllScheduledPods deletes all scheduled pods.
func (s *Service) DeleteAllScheduledPods(ctx context.Context) error {
```